using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -276,23 +276,31 @@ ACPIdentity.append(to:URL(string: "www.myUrl.com"), withCallback: {(appendedURL)
 {% tab title="React Native" %}
 #### JavaScript
 
-### appendToURL
+### appendVisitorInfoForURL<a id="appendToUrl-js"></a>
 
 Appends Adobe visitor information to the given URL.
 
 If the given url is nil or empty, it is returned as is. Otherwise, the following information is added to the query section of the given URL. The attribute `adobe_mc` is an URL encoded list containing the Experience Cloud ID, Experience Cloud Org ID, and a timestamp when this request was made. The attribute `adobe_aa_vid` is the URL encoded Visitor ID, however the attribute is only included if the Visitor ID was previously set.
 
+#### Syntax
+
 ```jsx
 ACPIdentity.appendVisitorInfoForURL(baseURL);
 ```
+#### Example
+
+```jsx
+ACPIdentity.appendVisitorInfoForURL("www.myUrl.com").then(urlWithVistorData => console.log("AdobeExperenceSDK: Url with Visitor Data = " + urlWithVisitorData));
+```
+
 {% endtab %}
 {% endtabs %}
 
 ## Get visitor data as URL query parameter  <a id="getUrlVariablesTitle"></a>
 
 {% tabs %}
 {% tab title="Android" %}
-### getUrlVariables
+### getUrlVariables<a id="geturlvariables-java"></a>
 
 _added in Identity v1.1.0_
```

Comment:
Let's update this to sdk-core 1.4.0 or add both versions (identity and sdk-core)